src/operators/repeat.ts

```diff
@@ -76,15 +72,15 @@ class MoreRepeatSubscriber<T> extends Subscriber<T> {
     super(null);
   }
 
-  _next(value: T) {
+  _next(value: T): void {
     this.parent.destination.next(value);
   }
 
-  _error(err: any) {
+  _error(err: any): void {
     this.parent.destination.error(err);
```

Comment:
This one also seems unreachable but I wasn't completely sure. @staltz , would like to ask opinions, is there usecases of `FirstRepeatSubscriber` doesn't raises error than `MoreRepeatSubscriber` does?